### DISCUSSION TITLE
Rofi -theme-str option doesn't work

### DISCUSSION THREAD

**Discussion by stibinator** [i3/i3#6326] [Q&A]
I'm using the rofi-power-menu to do my power menu, and I'm trying to make it a different style to my rofi application switcher menu. From the example given [here](https://github.com/jluttine/rofi-power-menu/tree/master) I can get it working how I like from a command line like this:
`~  rofi -show p -modi 'p:rofi-power-menu' -font 'Fira Sans 18' -theme-str 'window {width: 12em; height: 0;} listview {lines: 6;}'` 
but if I include it in my i3 config like this 
`bindsym $mod+Pause exec --no-startup-id rofi -show p -modi 'p:rofi-power-menu' -font 'Fira Sans 18' -theme-str 'window {width: 12em; height: 0;} listview {lines: 6;}'`

I get this error:
```ERROR: Received ExposeEvent for unknown window 0x0040001a
ERROR: Received ConfigureNotify for unknown window 0x0040001a
ERROR: Expected one of these tokens: <end>, '[', 'move', 'exec', 'exit', 'restart', 'reload', 'shmlog', 'debuglog', 'border', 'layout', 'append_layout', 'workspace', 'focus', 'kill', 'open', 'fullscreen', 'sticky', 'split', 'floating', 'mark', 'unmark', 'resize', 'rename', 'nop', 'scratchpad', 'swap', 'title_format', 'title_window_icon', 'mode', 'bar', 'gaps'
ERROR: Your command: exec --no-startup-id rofi -show p -modi 'p:rofi-power-menu' -font 'Fira Sans 18' -theme-str 'window {width: 12em; height: 0;} listview {lines: 6;}'
ERROR:                                                                                                                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/run/user/1000/i3/errorlog.43491 (END)
```

I'm not 100% sure of the rofi syntax, can someone explain what I'm doing wrong. My workaround is to put the rofi command in a shell script, but that seems unnecessarily kludgy.

**Comment by orestisfl** [comment]
Try
```
bindsym $mod+Pause exec --no-startup-id "rofi -show p -modi 'p:rofi-power-menu' -font 'Fira Sans 18' -theme-str 'window {width: 12em; height: 0;} listview {lines: 6;}'"
```

**Comment by stibinator** [comment]
Brilliant thanks. I'm unclear about how I could pass some parameters like the font without quoting the whole thing but not others. At least I know I can pass the whole command as a quoted string.
